share/qt/Info.plist.in

```diff
@@ -38,15 +38,15 @@
   <true/>
 
   <key>CFBundleIdentifier</key>
-  <string>org.bitcoinfoundation.Bitcoin-Qt</string>
+  <string>org.bitcoincore.Bitcoin-Qt</string>
```

Comment:
I think this could have further consequences... though not sure.
* codesigning?
* internal paths for preferences/settings?

Reply:
This change is dangerous. It will wipe out the settings
https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleidentifier

Reply:
Yes - there are "consequences" to the changes proposed in this PR. The long term benefits IMO out weigh the short term transition with the goals of this PR. More explanations soon.